freespace2/freespace.cpp

```diff
@@ -1376,11 +1389,15 @@ bool game_start_mission()
 
 		return false;
 	}
-	load_mission_load = (uint) (time(nullptr) - load_mission_load);
 
-	// free up memory from parsing the mission
-	extern void stop_parse();
-	stop_parse();
```

Comment:
Just wondering why this call was removed?

Is `stop_parse()` called elsewhere as well? Or is calling it not strictly required?